src/main/java/io/logz/logback/LogzioSender.java

```diff
@@ -339,13 +347,15 @@ private void debug(String message, Throwable e) {
     private String formatMessage(ILoggingEvent loggingEvent) {
 
         JsonObject logMessage = formatMessageAsJson(loggingEvent.getTimeStamp(), loggingEvent.getLevel().levelStr,
-                loggingEvent.getFormattedMessage(), loggingEvent.getLoggerName(), loggingEvent.getThreadName(), Optional.ofNullable(loggingEvent.getThrowableProxy()));
+                loggingEvent.getFormattedMessage(), loggingEvent.getLoggerName(), loggingEvent.getThreadName(),
```

Comment:
That was the original code.
I had to convert it to this because the convert method of Logback only accept ILoggingEvent and not Throwable (that method is protected)